File: PYTHON/pythonprograms/octal to decimal.py
n=int(input("enter a octal number: ")) 
dec_num=0
list1=list(str(n)) 
for j in range(len(list1)):
    a=(8**(j))*int(list1[(len(list1)-1)-j])
    dec_num+=a
print(dec_num) 

File: Abhinav Varshney 221B013/Number Conversions/octalToDecimal.c
#include <stdio.h>
int main()
{
    int b,d=0,base=1,temp;
    printf("Enter the octal number: ");
    scanf("%d",&b);
    int num=b;
    while(num!=0)
    {
        temp=num%10;
        d+=temp*base;
        base*=8;
        num/=10;
    }
    printf("The decimal for octal %d is %d",b,d);
    return 0;
}